native_client/alphabet.h

```diff
@@ -0,0 +1,46 @@
+#ifndef ALPHABET_H
+#define ALPHABET_H
+
+#include <cassert>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+
+/*
+ * Loads a text file describing a mapping of labels to strings, one string per
+ * line. This is used by the decoder, client and Python scripts to convert the
+ * output of the decoder to a human-readable string and vice-versa.
+ */
+class Alphabet {
+public:
+  Alphabet(const char *config_file) {
+    std::ifstream in(config_file, std::ios::in);
+    int label = 0;
+    for (std::string line; std::getline(in, line); ++label) {
```

Comment:
What do we do when we have duplicates ? Also, should we consider an empty line (only '\n') as valid ?

Reply:
Undefined behavior :P

I'll add a comment to the file defining the proper syntax, and prescribe that there shouldn't be duplicates. (As well as the line ending issue).